src/modules/meteor/index.js

```diff
@@ -104,7 +104,10 @@ export function push(api) {
           sslConfig: config.ssl,
           logConfig: config.log,
           volumes: config.volumes,
-          image: config.dockerImage || 'meteorhacks/meteord:base'
+          docker: Object.assign({
+            image:'meteorhacks/meteord:base',
```

Comment:
Hi @marcusrognes 
With a small tweak we can keep mup compatible with older config files. Can you check whether `config.dockerImage` exists and use it. If we do it in the `mup-api.js` file [here](https://github.com/marcusrognes/meteor-up/blob/5bba2e14ed888436ef659a589c9df076d1e9319d/src/mup-api.js#L18) we don't have to worry about it anywhere else.

```js
// NOTE: backward compatibility
if (!this.config.docker && this.config.dockerImage) {
  this.config.docker = {image: config.dockerImage};
  delete this.config.dockerImage;
}
```

Thanks